Gentoo Bug #136666

As input argument #3 is used as an address, fill it in with an address.

Danny van Dyk <kugelfang@gentoo.org>
--- src/engines/common/Resampler.h.orig	2006-11-26 17:57:48.000000000 +0100
+++ src/engines/common/Resampler.h	2006-11-26 17:58:08.000000000 +0100
@@ -96,7 +96,7 @@
                         "movswl   6(%0,%%edi,2), %%edx   # load sample 3\n\t"
                         "cvtsi2ss  %%edx, %%xmm2         # convert to float\n\t"
                         "shufps    $0x1b, %%xmm2, %%xmm2 # swap to correct order\n\t"
-                        :: "r" (pSrc), "r" (Pos), "m" (__4f)
+                        :: "r" (pSrc), "r" (Pos), "m" (&__4f)
                         :  "%eax", "%edx", "%edi"
                     );
                 }
@@ -137,7 +137,7 @@
                         "movswl  14(%0, %%edi,4), %%edx  # load sample 3 (right)\n\t"
                         "cvtsi2ss  %%edx, %%xmm3         # convert to float\n\t"
                         "shufps    $0x1b, %%xmm3, %%xmm3 # swap to correct order\n\t"
-                        :: "r" (pSrc), "r" (Pos), "m" (__4f)
+                        :: "r" (pSrc), "r" (Pos), "m" (&__4f)
                         :  "%eax", "%edx", "%edi"
                     );
                 }
